### DISCUSSION TITLE
Can .shadervariantlist use macro's?

### DISCUSSION THREAD

**Discussion by ohmaya** [o3de/o3de#12169] [Help Wanted]
In EnhancedPBR_ForwardPass_EDS.shadervariantlist, all the variants are based on options.

**Comment by siliconvoodoo** [comment]
I don't believe that file supports any preprocessor. In my understanding it's meant to be autogenerated by the "shader management console" which is a (GUI) tool all by itself.

**Reply by santorac** [reply]
That's correct, it only supports shader options.
The .shadervariantlist is an area of continued research and experimentation. If you would like to join in the conversation, we would welcome your contribution of ideas and use cases as we try to solve problems around shader variant management. These tickets are examples of our ongoing work:

https://github.com/o3de/o3de/issues/11792
https://github.com/o3de/o3de/issues/11380
https://github.com/o3de/o3de/issues/11383
https://github.com/o3de/o3de/issues/11388
https://github.com/o3de/o3de/issues/11388
